{
  "error" : {
    "root_cause" : [
      {
        "type" : "resource_already_exists_exception",
        "reason" : "index [twitter/lX1li8j5QUaj6BqgOQNV2A] already exists",
        "index_uuid" : "lX1li8j5QUaj6BqgOQNV2A",
        "index" : "twitter"
      }
    ],
    "type" : "resource_already_exists_exception",
    "reason" : "index [twitter/lX1li8j5QUaj6BqgOQNV2A] already exists",
    "index_uuid" : "lX1li8j5QUaj6BqgOQNV2A",
    "index" : "twitter"
  },
  "status" : 400
}
